### PR TITLE
Add eslint into scripts and fix eslint errors

### PR DESCRIPTION
**Pull request by igolopolosov**
Hello 👋

I'd like to add my small contribution to the project you are doing 😀
It's not changing any functionality or deployment pipeline. It's just fixing several linting problems.

My PR contains a few small fixes:
- [x] Start running eslint for entire project. 🏃 
- [x] Fix existing eslint issues. ⚠️ 

Please let me know what you think or make suggestion to improve this PR 👍

### CODE REVIEW

**Comment by abbottry** [comment]
Closing due to inactivity.

**Comment by igolopolosov** [comment]
Hello @abbottry could you please help me to resolve the issues?
GitHub played tricks with me not sending notifications :)

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -11,6 +11,7 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+// eslint-disable-next-line no-unused-vars
```
[the remark this answers is not in the document]
Comment (igolopolosov) [reply cypress/plugins/index.js]: @abbottry that's fair point, but I guess it's valid disabling in this situation.
Ideally you want to lint all the js files, no matter what it is: code, tests, configs. 
When you start excluding files from linting you start adding rules which can exclude files that needs to be linted 😁 
I would say it's better to keep linting as much as you can.
Disabling lint isn't bad thing if you do it for reason 👍

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "start": "NODE_ENV=development webpack-dev-server --open --config static.webpack.config.dev.js",
     "start-hot": "NODE_ENV=development webpack-dev-server --open --hot --config static.webpack.config.dev.js",
     "test": "NODE_OPTIONS=--max_old_space_size=4096 jest",
+    "lint": "eslint .",
```
[the remark this answers is not in the document]
Comment (igolopolosov) [reply package.json]: @abbottry Great point! As an external contributor I'm not able to see you process and I'm not sure I can change it. In my mind this PR is very first step to add linting. Once you are happy with rules you make next PR to integrate this in the process. 🚀 

I'm happy if you will help me to add linting to appropriate stage (build, pre-commit, deployment), what's the best place to add it from your point of view?

File: sharedUtils/config.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import config from '../static.config.json'
 import { cmrEnv } from './cmrEnv'
-import secretConfig from '../secret.config.json'
+import secretConfig from '../secret.config.json' // eslint-disable-line import/no-unresolved
```
[the remark this answers is not in the document]
Comment (igolopolosov) [reply sharedUtils/config.js]: Thank you @abbottry This file is not presented when you clone repo. so I think when you want to execute pure linting on the pipeline then the linting will fail.

If you want to remove this linting line it means that you need to execute some additional code before you start linting 🐳 It sounds like inappropriate coupling. Typically you execute linting for raw codebase and not against generated or copied file.

I would like to see your opinion on this point 😀

File: static/src/js/containers/MetricsEventsContainer/__tests__/MetricsEventContainer.test.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import Enzyme, { mount } from 'enzyme'
 import Adapter from 'enzyme-adapter-react-16'
-import $ from 'jquery'
+import $ from 'jquery' // eslint-disable-line no-unused-vars
```
[the remark this answers is not in the document]
Comment (igolopolosov) [reply static/src/js/containers/MetricsEventsContainer/__tests__/MetricsEventContainer.test.js]: @abbottry Fair point and I think you may consult me if this one is unused or ignored as you know the codebase more. Can you highlight should it be removed?

**Pull request closed without merging**